README.md

```diff
@@ -471,45 +459,46 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
 * When defining binary operators, name the parameter `other`(`<<` and `[]` are
   exceptions to the rule, since their semantics are different).
 
-* The names of predicate methods (methods that return a boolean value) should
-  end in a question mark (i.e. `Array#empty?`). Methods that don't return a
-  boolean, shouldn't end in a question mark.
+* Name predicate methods with a question mark. Predicate methods are methods
```

Comment:
I'd replace the `question mark` to  `?` . In other part of this document, characters such as slash, brackets etc... aren't written literally